README.md

```diff
@@ -345,47 +346,397 @@ clay export domain.com/_lists > lists.clay
+
+#### Arguments
+
+* `-w, --watch` enables watching of source files after compilation
+* `-m, --minify` enables minification and bundling of source files
+* `-i, --inlined` enables the generation of base64 inlined font CSS
+* `-l, --linked` enables the generation of linked font CSS
+* `-p, --plugins` allows running additional postcss plugins when compiling styles
+* `-g, --globs` allows compiling additional JavaScript to `public/js/_global.js`
+* `-r, --reporter` allows specifying how results should be logged
+
+#### Examples
+
+```bash
+# compile all assets once
+clay compile
+
+# compile and watch all assets
+clay compile --watch
+
+# compile all assets once for production environments
+clay compile --minify
+
+# compile all assets, creating both inlined and linked font CSS
+clay compile --inlined --linked
+```
+
+### Media
+
+```bash
+clay compile media [--watch] [--reporter <reporter>]
+```
+
+Copy component, layout, styleguide, and site-specific media files from their source folders to the `public` directory. Media files are images (`jpg`, `jpeg`, `png`, `gif`), `svgs`, and favicons (`ico`).
+
+* `components/<name>/media/` are referenced by component templates and get copied to `public/media/components/<name>/`
+* `layouts/<name>/media/` are referenced by layout templates and get copied to `public/media/layouts/<name>/`
+* `styleguides/<name>/media/` are referenced by that styleguide's CSS and get copied to `public/media/stylesguides/<name>/`
+* `sites/<name>/media/` are favicons and other site-specific icons that are referenced by particular components in the `<head>` of pages. They get copied to `public/media/sites/<name>/`
+
+#### Arguments
+
+* `-w, --watch` enables watching of source files after compilation
+* `-r, --reporter` allows specifying how results should be logged
+
+#### Examples
+
+```bash
+# compile media files once
+clay compile media
+
+# compile and watch media files
+clay compile media --watch
+```
+
+### Fonts
+
+```bash
+clay compile fonts [--watch] [--minify] [--inlined] [--linked] [--reporter <reporter>]
+```
+
+Compile fonts from `styleguides/<name>/fonts/` to the `public` directory. By default (and if `--linked` is specified or `CLAYCLI_COMPILE_LINKED_FONTS` is set), this will generate a `public/css/_linked-fonts.<name>.css` file with `@font-face` declarations and copy the original font file to `public/fonts/`. Note that naming collisions are possible when using fonts of the same filename across different styleguides. If `--inlined` is specified (or `CLAYCLI_COMPILE_INLINED_FONTS` is set), this will generate a `public/css/_inlined-fonts.<name>.css` file with `@font-face` declarations that include a base64-encoded copy of the font.
```

Comment:
Just checking I have this right:
`clay compile fonts` = linked
`clay compile fonts --linked` = linked
`clay compile fonts --inlined` = inlined
`clay compile fonts --inlined --linked` = linked + inlined

Reply:
Yeah